definitions/divorce/json/AuthorisationCaseState.json

```diff
@@ -1334,7 +1334,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseStateID": "AwaitingLegalAdvisorReferral",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "R"
```

Comment:
Yeah this should be `CRU` as the solicitor will move the case to this state after DN submission

Reply:
I agree with @alexwakeman. We have the DN event in prod now so need to keep this as CRU otherwise will probably produce an error when solicitor tries to submit DN application.

Reply:
Done 👍